samples/snippets/conftest.py

```diff
@@ -50,6 +50,14 @@ def dataset_id(bigquery_client: bigquery.Client, project_id: str):
     bigquery_client.delete_dataset(dataset, delete_contents=True, not_found_ok=True)
 
 
+@pytest.fixture(scope="session")
+def entity_id(bigquery_client: bigquery.Client, dataset_id: dataset_id):
```

Comment:
Typo in type annotations?

```suggestion
def entity_id(bigquery_client: bigquery.Client, dataset_id: str):
```